packages/frontend/src/components/screens/WelcomeScreen/index.tsx

```diff
@@ -41,7 +41,7 @@ export default function WelcomeScreen({ selectedAccountId, ...props }: Props) {
 
   return (
     <ImageBackdrop variant='welcome'>
-      <Dialog fixed onClose={onClose} width={400}>
+      <Dialog fixed onClose={onClose} width={400} canOutsideClickClose={false}>
```

Comment:
```suggestion
      <Dialog fixed width={400} canOutsideClickClose={false}>
```

Also need to remove the `onClose` listener above.